build/ci/evergreen.yml

```diff
@@ -108,6 +108,22 @@ functions:
           TEST_CMD: gotestsum --junitfile e2e-tests.xml --format standard-verbose --
           <<: *go_options
         command: make e2e-test
+  "e2e test ops manager":
+    - command: subprocess.exec
+      type: test
+      params:
+        add_to_path:
+          - ${go_bin}
+        include_expansions_in_env:
+          - MCLI_SERVICE
+          - TEST_CMD
+          - E2E_TAGS
+        working_dir: src/github.com/mongodb/mongocli
+        env:
+          XDG_CONFIG_HOME: ${workdir}
+          TEST_CMD: gotestsum --junitfile e2e-tests.xml --format standard-verbose --
+          <<: *go_options
+        command: make e2e-test
```

Comment:
I am wondering if I can reuse `"e2e test"` 🤔

Reply:
I am running a patch to see if I can. I will update accordingly once it has finished

Reply:
nope 😞 https://evergreen.mongodb.com/task/mongocli_master_e2e_generic_ops_manager_generic_4_2_e2e_patch_60349802f2b9d4a011e9f9f8f189cd2a814b0911_60183a5c3e8e861a46ea7f7f_21_02_01_17_29_10

Reply:
it's failing with an error running the command, what's the error?

Reply:
it was failing because `e2e test` was setting `MCLI_OPS_MANAGER_URL`